### PR TITLE
FloorMod operation specification refactoring.

### PR DESCRIPTION
**Pull request by jdanieck**
FloorMod operation specification refactoring. Ticket 50101.

Depends on #4366 which introduce broadcast_rules.md.

### CODE REVIEW

File: docs/ops/arithmetic/FloorMod_1.md [hunk 1]
```diff
@@ -4,41 +4,49 @@
...
+**Short description**: *FloorMod* performs an element-wise floor modulo operation with two given tensors applying broadcasting rule specified in the *auto_broadcast* attribute. 
+
+**Detailed description**
+As a first step input tensors *a* and *b* are broadcasted if their shapes differ. Broadcasting is performed according to `auto_broadcast` attribute specification. As a second step *FloorMod* operation is computed element-wise on the input tensors *a* and *b* according to the formula below:
+
+\f[
+o_{i} = a_{i} % b_{i}
+\f] 
+
+*FloorMod* operation computes a reminder of a floored division. It is the same behaviour like in Python programming language: `floor(x / y) * y + floor_mod(x, y) = x`. The sign of the result is equal to a sign of a dividend.
```
Comment (lazarevevgeny) [inline docs/ops/arithmetic/FloorMod_1.md:16]: May be it make sense to add a comment for case when the divisor is 0 that the output value is undefined.

File: docs/ops/arithmetic/FloorMod_1.md [hunk 2]
```diff
@@ -4,41 +4,49 @@
...
+**Short description**: *FloorMod* performs an element-wise floor modulo operation with two given tensors applying broadcasting rule specified in the *auto_broadcast* attribute. 
+
+**Detailed description**
+As a first step input tensors *a* and *b* are broadcasted if their shapes differ. Broadcasting is performed according to `auto_broadcast` attribute specification. As a second step *FloorMod* operation is computed element-wise on the input tensors *a* and *b* according to the formula below:
+
+\f[
+o_{i} = a_{i} % b_{i}
+\f] 
+
+*FloorMod* operation computes a reminder of a floored division. It is the same behaviour like in Python programming language: `floor(x / y) * y + floor_mod(x, y) = x`. The sign of the result is equal to a sign of a dividend. Division by zero is undefined.
```
Comment (lazarevevgeny) [inline docs/ops/arithmetic/FloorMod_1.md:16]: The result of division by zero is undefined.

**Review by lazarevevgeny — APPROVED** [review]

File: docs/ops/arithmetic/FloorMod_1.md (hunk 2 above)
Comment (sdurawa) [reply docs/ops/arithmetic/FloorMod_1.md]: Fixed.

**Comment by sdurawa** [comment]
@lazarevevgeny , this PR is ready to merge.

**Pull request merged**